src/sentry/notifications/defaults.py

```diff
@@ -36,5 +36,6 @@
 
 NOTIFICATION_SETTING_DEFAULTS = {
     ExternalProviders.EMAIL: NOTIFICATION_SETTINGS_ALL_SOMETIMES,
-    ExternalProviders.SLACK: NOTIFICATION_SETTINGS_SLACK,
+    ExternalProviders.SLACK: NOTIFICATION_SETTINGS_INSTANT_MESSAGING,
+    ExternalProviders.MSTEAMS: NOTIFICATION_SETTINGS_INSTANT_MESSAGING,
```

Comment:
@97varun  So we decided to make MSTeams disabled by default?

Reply:
To start with I think yes, I'm keeping it the same as how it would be for Slack.

Reply:
@97varun So we made a change to make all new users have Slack by default. I need to backfill the old accounts before I can change the default, however.